refined/src/main/scala/magnolify/refined/package.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package magnolify
+
+import eu.timepit.refined._
+import eu.timepit.refined.api._
+import magnolify.avro.AvroField
+import magnolify.bigquery.TableRowField
+import magnolify.bigtable.BigtableField
+import magnolify.datastore.EntityField
+import magnolify.protobuf.ProtobufField
+import magnolify.tensorflow.ExampleField
+
+import scala.language.implicitConversions
+
+package object refined {
+  object avro {
+    implicit def refinedAvro[T: AvroField, P](implicit v: Validate[T, P]): AvroField[Refined[T, P]] =
+      AvroField.from[T](unsafeRefine[T, P])(_.value)
+  }
+
+  object bigquery {
```

Comment:
is this safe if `magnolify-bigquery` is not in the classpath?

Reply:
Yeah all dependencies are `Provided` so people will have to explicitly include `magnolify-bigquery` etc. This is so that we don't create a `magnolify-refined-*` for each one of Avro, BQ, Bigtable, etc.